x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/index.tsx

```diff
@@ -22,17 +24,28 @@ import { AllRulesTabs, RulesTableToolbar } from './rules_table_toolbar';
  *   * Import/Export
  */
 export const AllRules = React.memo(() => {
-  const [activeTab, setActiveTab] = useState(AllRulesTabs.rules);
-
   useInitializeRulesTableSavedState();
   useSyncRulesTableSavedState();
 
   return (
     <>
       <RulesManagementTour />
-      <RulesTableToolbar activeTab={activeTab} onTabChange={setActiveTab} />
-      <EuiSpacer />
-      <RulesTables selectedTab={activeTab} />
+      <Switch>
+        <Route path="/rules/monitoring">
```

Comment:
```suggestion
        <Route path="/rules/:tabName(${AllRulesTabs.monitoring})">
```

Reply:
I eventually got rid of `Switch` component here since having it only in `routes.ts` is enough.